### PR TITLE
Fixes wormhole projectors for pacifists

### LINKED ISSUE #43054: Pacifists can't use wormhole guns
P sure wormhole guns can't hurt people with portals now, but they still can't be shot. ye

### PR DESCRIPTION
**Pull request by ShizCalev**
Fixes #43054

:cl: ShizCalev
fix: Pacifists can now use wormhole projectors.
/:cl:

### CODE REVIEW

**Comment by SouDescolado** [comment]
I'm not sure about areas and turfs, but could it blacklist turfs because it could be dangerous? Like space or something? Otherwise I think this will be the go to weapon for Pacifist Traitors (I think @imsxz made a PR that got merged which allows this)

**Comment by HomocidlePancakes** [comment]
It's super easy to harm someone with a gravity manipulator. You shouldn't allow pacifists to use  it

**Comment by Ratvar** [comment]
Same way it's easy to harm someone using wormhole projectors. They can't shoot syringe guns, why let them shoot those?

**Comment by Cyprex** [comment]
A lot of things can be abused as a pacifist, with the variety of things you can do in this game there will always be loopholes. Restricting half the game from pacifists doesn't seem like the right way.

**Comment by Ratvar** [comment]
If you don't want to be a pacifist, don't be. Or use some more (then a gun) creative damage dealer. Wormhole projectors and gravity manipulators (and guns) are in no way "half of the game".

**Comment by SouDescolado** [comment]
> Or use some more (then a gun) creative damage dealer.

Or do no damage. You are a pacifist after all.

**Comment by Ratvar** [comment]
Sadly, that wouldn't be feasable because you can't code all that forbidding shit. Can forbid guns tho.

**Comment by dr-bee** [comment]
Wormhole projectors makes sense, but the gravity gun is a literal weapon. it does damage by throwing shit at the target.

**Comment by ExcessiveUseOfCobblestone** [comment]
wormhole has nonantag uses, gravity is just a mislabeled griefgun

**Comment by Ratvar** [comment]
Syringe gun can be used for healing people, so? They are still killing weapons and will be used for killing by pacifists without a second thought

**Comment by GuyonBroadway** [comment]
Literally what is the point of the gravity gun?

Maybe if you could move normally unmoveable shit around with it, it's be neato, I know atmos techs would sick a dick or two to have a gun that lets you move heaters/freezers without needing to deconstruct the dang things.

**Comment by imsxz** [comment]
grav gun is for killing humanoid antags that dare show up that long into a round (generally a free short stun on everyone in the massive AOE) and or running around shooting it for epic grief

I used a grav gun to dislodge an item that I couldn’t reach but otherwise I’d be impressed if someone managed to make use of one without hurting someone in the process. Probably a really bad idea for pacifists.

While wormhole guns do have some creative uses to kill people as pacifist, its primary purpose is harmless and very nice to have for whatever a science powergamer may have in mind I don’t really need to go into detail

File: code/modules/mob/living/simple_animal/hostile/jungle/mega_arachnid.dm [hunk 1]
```diff
@@ -50,7 +50,7 @@
 
 /obj/item/projectile/mega_arachnid
 	name = "flesh snare"
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/mob/living/simple_animal/hostile/jungle/mega_arachnid.dm]: lewd

File: code/modules/projectiles/ammunition/energy/portal.dm [hunk 2]
```diff
@@ -1,9 +1,10 @@
 /obj/item/ammo_casing/energy/wormhole
 	projectile_type = /obj/item/projectile/beam/wormhole
 	e_cost = 0
+	harmful = FALSE
 	fire_sound = 'sound/weapons/pulse3.ogg'
-	var/obj/item/gun/energy/wormhole_projector/gun = null
 	select_name = "blue"
+	var/obj/item/gun/energy/wormhole_projector/gun = null
```
Comment (ninjanomnom) [inline code/modules/projectiles/ammunition/energy/portal.dm]:
```suggestion
	var/obj/item/gun/energy/wormhole_projector/gun
```

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

**Pull request merged**